woocommerce/woocommerce-android
# Issue/4662 New way of connecting to reader

This PR implements passing of the location id which comes from simulated reader

Closes: #4662

### Description
The PR makes it possible to connect to a simulated reader

### Testing instructions
* Open an order eligible for IPP
* Collect payment
* Notice that it does collect payment on the SDK 2.1.0

### Images/gif

https://user-images.githubusercontent.com/4923871/130578988-bc254813-7c9f-45e1-8cd8-a763b1571dba.mp4
